### ISSUE TITLE
Controller support improvements

### ISSUE THREAD

**Issue by msgerbs** [DonBruce64/MinecraftTransportSimulator#766]
The controller config for this mod seems designed for a wheel and pedal for driving or a flight stick and throttle for flying, and doesn't lend itself well to a standard dual analog stick controller with trigger buttons. Here are some issues I ran into and what I would suggest to fix them:

For driving:

1. Throttle and brake are separate axes, both centered at -1 (or whatever the lowest position of the axis is). This doesn't work well if trying to use an analog stick for gas/brake because the stick centers at 0 and so centers at 50% throttle or brake and must be held down to be at 0% throttle. To fix this, an option for a combined gas/brake axis which is neutral at 0 or whatever the center of the stick's range is would work.
2. There's no gas button, only a brake button. This makes using button triggers for gas/brake like most driving games impossible. To fix this, add a button for gas like already exists for brakes.

For flying:

1. Same issue with throttle, it is at 0% when the axis is at its lowest position. This works great with an actual throttle lever but not with an analog stick. An option to again combine brakes and throttle would work here as well.
2. There are no buttons for throttle at all, only brakes. This again makes using triggers to control throttle impossible. Adding a ThrottleUp and ThrottleDown button would fix this nicely. I don't think a simple "gas" button would work well here since you don't usually just want all or nothing like with a car, but it might be nice to have anyway if it's not too much extra work.
3. Add buttons for YawLeft and YawRight. This one isn't really a problem per se, but flying games that use a controller often bind YawLeft and YawRight to the triggers since you don't really need these controllers very often and usually not with very much precision. Adding buttons for this would make it easier to emulate a a more traditional controller config.

**Comment by msgerbs** [comment]
Regarding the throttle, I found a config that works but it was rather confusing to set up. I had to only move the stick forward during calibration for the gas, and invert it. And then move it only backwards for the brake and not invert it. And I had to move the stick slightly backward before calibrating the brake to ensure there was absolutely no overlap between the 2. Anything else resulted in very strange conflicts where both pedals would get pressed at the same time even though I had only moved it in one direction when calibrating it. So that complaint may be invalid, but documentation improvements there would help a lot. The rest of these still seem valid.
